src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Android/AndroidInstallCommandArguments.cs

```diff
@@ -44,15 +44,22 @@ protected override OptionSet GetTestCommandOptions() => new()
                 v => DeviceId = v
             },
             {
-                "boot-timeout=", "Timeout in seconds to wait for a device after boot completion",
-                v => {
-                    if (int.TryParse(v, out var number))
+                "launch-timeout=|lt=", "TimeSpan or number of seconds to wait for a device after boot competion",
```

Comment:
I don't understand this `after completion`. We are waiting **for** the completion, right?

Reply:
We are waiting for _boot_ completion, checking device status with command `getprop sys.boot_completed`

Reply:
Right. So I think it is a bit confusing `wait for a device after boot competion`?

I would say: `... to wait for the device to boot to complete`?